modules/auxiliary/admin/sap/sap_igs_xxe.rb

```diff
@@ -0,0 +1,423 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+# Yvan @_1ggy Genuer
+# Vladimir @_generic_human_ Ivanov
```

Comment:
These lines should be removed as these should be included in the `Author` section. If you want to add your Twitter handles in, then please put them as comments next to the author names